modules/arrow/test/geoarrow/convert-geoarrow-to-geojson.spec.ts

```diff
@@ -281,30 +279,30 @@ async function testParseFromArrow(
   arrowFile: string,
   expectedGeojson: FeatureCollection
 ): Promise<void> {
-  // TODO: use the following code instead of apache-arrow to load arrow table
-  // const arrowTable = await parse(fetchFile(arrowFile), ArrowLoader, {worker: false});
-  const response = await fetchFile(arrowFile);
-  const arrayBuffer = await response.arrayBuffer();
-  const arrowTable = tableFromIPC(new Uint8Array(arrayBuffer));
-
-  t.comment(arrowFile);
+  const arrowTable = await parse(fetchFile(arrowFile), ArrowLoader, {
+    worker: false,
+    arrow: {
+      shape: 'arrow-table'
+    }
+  });
 
+  const table = arrowTable.data as arrow.Table;
```

Comment:
Nit: it is possible to avoid the cast by using the discriminated union that is returned.
```
if (arrowTable.shape === 'arrow-table) {
  arrowTable.data; // (cast not necessary)
```